docs/PacketFence_Administration_Guide.asciidoc

```diff
@@ -494,7 +494,9 @@ Roles Management
 
 Roles in PacketFence can be created from PacketFence administrative GUI - from the *Configuration -> Policies and Access Control -> Roles* section. From this interface, you can also limit the number of devices users belonging to certain roles can register.
 
-Roles are dynamically computed by PacketFence, based on the rules (ie., a set of conditions and actions) from authentication sources, using a first-match wins algorithm. Roles are then matched to VLAN or internal roles or ACL on equipment from the *Configuration -> Policies and Access Control -> Switches* module.
+Roles are dynamically computed by PacketFence, based on the rules (ie., a set of conditions and actions) from authentication sources, using a first-match wins algorithm. Roles are then matched to VLAN or VLAN Pool or internal roles or ACL on equipment from the *Configuration -> Policies and Access Control -> Switches* module.
+
+For the VLAN Pool instead of defining a Vlan ID you can set a vlaue like that : 20..23,27..30 , it mean that the vlan returned by PacketFence can be (inclusive)  20 to 23 and 27 to 30. There are two algorithm, one based on a has of the username (default one) and the other one based on a round robin (last registered device +1).
```

Comment:
Pool -> pool, Vlan -> VLAN, ID -> identifier, vlaue ->value, it mean -> it means, vlan -> VLAN, two algorithm -> two algorithms, has of -> hash of, on a round robin -> on round-robin.